ialirt_data_access/cli.py

```diff
@@ -259,6 +272,12 @@ def main():
         required=False,
         help="End of last_modified time.",
     )
+    db_query_parser.add_argument(
+        "--downloads_dir",
+        type=Path,
+        required=False,
+        help="Example: /path/to/downloads/dir",
+    )
```

Comment:
I wonder if you can add a default here, and add that to the `help` output? Like, "Default is the user's Downloads folder."